### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by Purkinje-cell** [community/community#33344] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "Unexpected token N in JSON at position 0"

Can anyone tell me how to solve it? I'm using VSCode Copilot Plugins on Window 10

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by Purkinje-cell** [comment, marked as answer]
Solved it. I had modified my hosts file because I can't connect to github.After I restored it, I can use Copilot again.
